tests/platform_tests/sfp/test_sfpshow.py

```diff
@@ -57,4 +71,19 @@ def test_check_sfpshow_eeprom(duthosts, enum_rand_one_per_hwsku_frontend_hostnam
     for intf in dev_conn:
         if intf not in xcvr_skip_list[duthost.hostname]:
             assert intf in parsed_eeprom, "Interface is not in output of 'sfputil show eeprom'"
-            assert parsed_eeprom[intf] == "SFP EEPROM detected"
+            assert parsed_eeprom[intf] == "SFP EEPROM detected", (
+                "The EEPROM information for interface '{}' is not as expected. "
```

Comment:
Don't need to add those indications in failed reason, that's not related to the failed reason.
It's better to print the whole parsed_eeprom out.